app/services/post_scraper.rb

```diff
@@ -167,12 +168,14 @@ def import_post_from_doc(doc)
     end
 
     set_from_username(@post, username)
```

Comment:
`set_from_username` tries to set a character, which will fail on a `@post`, right? Should we `set_from_username(@written, username)` and then `@post.user = @written.user`?

Reply:
Oh. You check if it's a Post later. I guess.

Reply:
Huh, I wonder why the tests didn't catch that